provider/virt_storage/storage_volume.py

```diff
@@ -151,6 +156,17 @@ def refresh_protocol_by_params(self, params):
             aio = params.get("image_aio", "threads")
             self.protocol.set_param("filename", self.path)
             self.protocol.set_param("aio", aio)
+        elif self.protocol.TYPE == "rbd":
+            self.protocol.set_param("pool", self.pool.name)
+            image_name = self.path.split("/")[-1]
+            self.protocol.set_param("image", image_name)
+            if self._params.get("ceph_monitor"):
+                servers = []
+                for item in self._params["ceph_monitor"].split():
+                    host, port = item.split(":")
+                    server = {"host": host, "port": port}
+                    servers.append(server)
+                self.protocol.set_param("server", servers)
```

Comment:
we can remove these lines, because the server param is not required, for rbd, only rbd pool name and image name are required

Reply:
Have removed these lines, thanks zhenchao